src/inputs/pipe.c

```diff
@@ -39,6 +39,7 @@
 #include <string.h>
 #include <errno.h>
 #include <limits.h>
+#include <math.h>
```

Comment:
Yeap I thought about it as well it would work without math and without the extra link option by not using round and abs.